src/FastBoard.cpp

```diff
@@ -255,17 +268,30 @@ bool FastBoard::is_suicide(int i, int color) {
     }
 }
 
+/*
+    Liberties on a single point - called pseudo liberties.
+    When placing stones adjacent to that group, add one pseudo liberty for neighbouring empty cells,
+    and remove one pseudo liberty for each neighboring group.
+    There may be duplicate pseudo liberties with certain "bent" shapes.
+    However, the test for 0 liberties remains correct, and there is also a simple "atari" check.
+    @return number of pseudo liberties
+*/
 int FastBoard::count_pliberties(const int i) {
     return count_neighbours(EMPTY, i);
 }
 
-// count neighbours of color c at vertex v
-// the border of the board has fake neighours of both colors
+/*
+    Count neighbours of color c at vertex v.
+    The border of the board has fake neighours of both colors.
+*/
 int FastBoard::count_neighbours(const int c, const int v) {
     assert(c == WHITE || c == BLACK || c == EMPTY);
     return (m_neighbours[v] >> (NBR_SHIFT * c)) & 7;
 }
 
+/* 
+    @return true if playing the specified single stone (ss) is suicidal. 
```

Comment:
true if playing at the given location is a single-stone suicide.

Reply:
Updated the comment to your suggestion.